packages/ffe-form-react/src/PhoneNumber.tsx

```diff
@@ -11,6 +11,8 @@ export interface PhoneNumberProps {
     onNumberChange?: React.ChangeEventHandler<HTMLInputElement>;
     onCountryCodeBlur?: React.FocusEventHandler<HTMLInputElement>;
     onNumberBlur?: React.FocusEventHandler<HTMLInputElement>;
+    countryCodeRequired?: boolean;
```

Comment:
man kunde ju kanskje vurdert
```
numberInputProps: React.ComponentPropsWithoutRef<'input'>;
countryInputProps: React.ComponentPropsWithoutRef<'input'>;
````

Da man kan man slette en den av propsen her.  Ville blitt en BREAKING CHANGE. Men dette er ju i linje med det andre i denne  filen. Føljer dette er designsystemet fula lilla ankunge eftersom den ikke kan gå i en Inputgroup som alla annat

Reply:
Gidder du ordne `this should should ` i samme fila?
``

Reply:
hmm det var jo lurt da, jeg tror jeg gjør det